internal/driver/glfw/window.go

```diff
@@ -115,7 +115,8 @@ func (w *window) SetFullScreen(full bool) {
 		if full {
 			w.viewport.SetMonitor(monitor, 0, 0, mode.Width, mode.Height, mode.RefreshRate)
 		} else {
-			w.viewport.SetMonitor(nil, w.xpos, w.ypos, w.width, w.height, 0)
+			w.viewport.SetMonitor(nil, w.xpos, w.ypos, w.size.Width, w.size.Height, 0)
+			w.fullScreen = false
```

Comment:
Isn't this set in line 106?

Reply:
Oops, this wasn't there before merging with develop, will fix.